### ISSUE TITLE
elasticache: cannot create users with passwords

### ISSUE THREAD

**Issue by matteo-magni** [upbound/provider-aws#151]
### What happened?

The latest provider-aws release (v0.20.0) cannot create Elasticache users with passwords set via `.spec.forProvider.passwordsSecretRef`.
Apparently such field is not translated into its terraform equivalent [here](https://github.com/upbound/provider-aws/blob/18593b3c9b2d73a5426dd4932445c64d8753f7aa/apis/elasticache/v1beta1/zz_user_types.go#L42).

### How can we reproduce it?

1. install and configure provider-aws
2. create the secret
   ```sh
   kubectl -n crossplane-system create secret generic user-passwords --from-literal pwd-1=mypass1 --from-literal pwd-2=mypass2
   ```
3. apply [this example](https://github.com/upbound/provider-aws/blob/v0.20.0/examples/elasticache/user.yaml) from the git repo or copy it and adjust the AWS region if needed:
   ```sh
   kubectl apply -f https://raw.githubusercontent.com/upbound/provider-aws/v0.20.0/examples/elasticache/user.yaml
   ```
4. observe the object's status:
   ```sh
   kubectl get users.elasticache.aws.upbound.io sample-user -o yaml
   ```
   The `LastAsyncOperation` condition reports
   ```yaml
   message: "apply failed: error creating ElastiCache User: InvalidParameterValue:
     Input Authentication type: null is not in the allowed list: [password,no-password-required,iam]\n\tstatus
     code: 400, request id: e98ac4d4-66e3-4ebb-839c-be68fdde2416: "
   reason: ApplyFailure
   status: "False"
   type: LastAsyncOperation
   ```
   which is the same error the AWS CLI returns when trying to create a user with no `--passwords` nor `--no-password-required`
   ```sh
   $ aws elasticache create-user --user-id sample-user --user-name sample-user --engine REDIS --access-string "on ~* +@all"
   
   An error occurred (InvalidParameterValue) when calling the CreateUser operation: Input Authentication type: null is not in the allowed list: [password,no-password-required,iam]
   ```

### What environment did it happen in?

* Universal Crossplane Version: v1.10.1-up.1
* Provider Version: v0.20.0
* Cloud provider: AWS
* Kubernetes distribution: EKS
* Kubernetes version: v1.23.13-eks-fb459a0

**Comment by turkenf** [comment]
The issue reproduced.

**Comment by turkenf** [comment]
The relevant upstream issue has been fixed and closed, will test this issue after upgrading the https://github.com/crossplane/upjet

**Comment by turkenf** [comment]
This issue has been fixed with Provider versions `0.47.3` and `1.1.1`. I am closing this now, feel free to reopen it if you encounter any issues.
